### DISCUSSION TITLE
How to remove a theme from github pages

### DISCUSSION THREAD

**Discussion by Benmuiruri** [community/community#21759] [Other Feature Feedback, Questions, & Ideas]
<p>I accidentally chose the Cayman theme for my github page at <a href="https://benmuiruri.github.io/Wangari-Maathai-Tribute/" rel="nofollow noopener">https://benmuiruri.github.io/Wangari-Maathai-Tribute/</a></p>
<p>I would like to disable the theme and use my own custom layout. </p>
<p>How do I go about disabling a theme / removing a theme and using my layout? </p>

**Comment by lee-dohm** [comment, marked as answer]
<p>This is essentially a duplicate of <a href="https://github.community/t/how-to-remove-theme-from-project/10454">a previous topic</a>. That topic will show you how to remove the theme. Once that’s done, you can follow the instructions in <a href="https://help.github.com/en/articles/customizing-css-and-html-in-your-jekyll-theme#customizing-your-jekyll-themes-html-layout" rel="nofollow noopener">the Pages help documents on how to customize your Pages site’s layout</a>.</p>
<p>I hope that helps!</p>
